magento/magento2
# Fix fulltext search of attributes with multiselect option values

### Description
Fixes https://github.com/magento/magento2/issues/13972
A bug was introduced in https://github.com/magento/magento2/commit/0ff69c5cbc85af82cd87371c6fe01718a6e463ac which tried optimizing
getting option labels but did not take into account
$valueId could be a comma separated string. This restores
the old behavior and the performance difference is negligible.
Bug only affected 2.2
I'm a little new to unit tests so feel free to give feedback on it, or we could scrap it all together.

### Manual testing scenarios
Create a custom product attribute with the multiselect input type. Create several options. Make the attribute searchable and shown in advanced search. 
Create product with the attribute and select several values.
Searching by label name should be possible in the quick search or advanced search. 


### Contribution checklist
